contrib/dockerd-rootless.sh

```diff
@@ -128,5 +128,23 @@ else
 		mount --rbind ${realpath_etc_ssl} /etc/ssl
 	fi
 
-	exec dockerd $@
+	dockerd="$(which dockerd)"
+
+	if [ -n "$DELVE_PORT" ]; then
+		echo "Exposing Delve debugger on port $DELVE_PORT"
+
+		# shellcheck disable=SC2086
+		# shellcheck disable=SC2068
+		exec dlv --listen=0.0.0.0:$DELVE_PORT \
+			--headless=true \
+			--log \
+			--api-version=2 \
+			--only-same-user=false \
+			--check-go-version=false \
+			--accept-multiclient \
+			exec "${dockerd}" -- $@
+	else
+		# shellcheck disable=SC2068
+		exec "${dockerd}" $@
+	fi
```

Comment:
I don't think we should have delve integration here.

The script should be modified to support an alternative `dockerd` path via `$DOCKERD`.

Reply:
- [ ] @1. [`**He*really*`](url)

Reply:
Good point, I’ll modify `dockerd-rootless.sh` to optionally use the `DOCKERD` variable and set it to `dlv` with arguments in the run script. This will also remove duplicate `dlv` executions.